exe/slow_query_exporter

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env ruby
+
+require 'daemons'
+require 'gelf'
+require 'optparse'
+require 'syslog'
+require 'slow_query_exporter'
+
+RUN_DIRECTORY = "/var/run/slow_query_exporter"
+PROGRESS_FILE = "#{RUN_DIRECTORY}/last_timestamp"
+
+$graylog_host = "localhost"
+$graylog_port = 12201
+$delay = 0.1
+$verbose = false
+$foreground = false
+
```

Comment:
Yes, I see your rational, especially given an executable script.  And yes, `honk` will be undefined in the method.